app/views/cyclists/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="content-wrapper-narrow content-container">
+  <header>
+    <h1><%= t(".header", name: @cyclist.name) %></h1>
```

Comment:
I'd go ahead and make this an `h2` or `h3`, since the logo in the header is marked up as the `h1` on this and other pages. (This is just for SEO)